src/ui/storyTellingServerUI.js

```diff
@@ -109,28 +109,16 @@ sjrk.storyTelling.loadBrowse = function (clientConfig, options) {
 };
 
 /* Gets the current theme from the server and loads associated  files if and
- * only if the current theme is not set to the base theme. Once complete, the
- * provided callback function is called and the theme name is passed into it.
- * - "callback": a function to call once everything has completed. Will be called
- *               regardless of whether theme information was specified or retrieved
+ * only if the current theme is not set to the base theme. Returns a promise.
+ * If the promise resolves, it will contain the clientConfig information.
  */
-sjrk.storyTelling.loadThemedPage = function (callback) {
+sjrk.storyTelling.loadTheme = function () {
     var loadPromise = fluid.promise();
 
-    var callbackFunction = typeof callback === "function" ? callback : fluid.getGlobalValue(callback);
-
     $.get("/clientConfig").then(function (data) {
```

Comment:
It would probably be a bit more readable/descriptive if the argument is called `clientConfig` instead of `data`.